### PR TITLE
Add input to backoffice

### PR DESCRIPTION
**Pull request by amiryonatan**

### CODE REVIEW

File: packages/wix-ui-backoffice/src/components/Input/theme.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+};
+
+export const theme = ({size, skin}) => ({
+
+  color: skinToColor[skin],
+  backgroundColor: palette.white,
+  borderColor: skinToBorderColor[skin],
+
+  height: sizeToHeight[size],
+  lineHeight: sizeToHeight[size] - 2,
```
Comment (lbelinsk) [inline packages/wix-ui-backoffice/src/components/Input/theme.js]: did you make sure this is working? check in the style tag in the head, I am afraid maybe only strings allowed

File: packages/wix-ui-core/src/components/Input/Input.tsx [hunk 2]
```diff
@@ -29,7 +29,7 @@ interface InputProps {
   readOnly: bool;
   required: bool;
   tabIndex: number;
-  type: string;
+  type: 'text' | 'number';
```
Comment (lbelinsk) [inline packages/wix-ui-core/src/components/Input/Input.tsx]: type is a core attribute. It shouldn't be specific to the backoffice

File: packages/wix-ui-core/src/components/Input/theme.ts [hunk 3]
```diff
@@ -38,22 +46,29 @@ export const core: InputTheme = {
   textDecoration: 'none',
 
   height: '36px',
-  padding: '0 23px',
+  padding: '0 12px',
```
Comment (lbelinsk) [inline packages/wix-ui-core/src/components/Input/theme.ts]: please find a time (not a blocker) to get Ben's review. when you feel it is time

**Comment by lbelinsk** [comment]
Don't forget about the e2e's

File: packages/wix-ui-core/src/components/Input/Input.e2e.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+  beforeEach(() => {
+    browser.get(storyUrl);
+  });
+
+  eyes.it('should enter text to inpux', () => {
+    const dataHook = 'story-button-enabled';
+    const driver = inputTestkitFactory({dataHook});
+
+    waitForVisibilityOf(driver.element(), 'Cannot find Input')
```
Comment (lbelinsk) [inline packages/wix-ui-core/src/components/Input/Input.e2e.ts]: @argshook fyi
If you will add a new wrapper to run e2e tests just remember to modify this as this is the first e2e in the lib

Comment (argshook) [reply packages/wix-ui-core/src/components/Input/Input.e2e.ts]: i wont add new wrapper, should be possible to reuse `AutoExampleDriver` for components documented with `story`, but it's optional. However, i still need to play with it and document its usage

File: packages/wix-ui-core/protractor.conf.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  baseUrl: `http://localhost:6006/`,
+  specs: ['test/**/*.e2e.ts', 'src/**/*.e2e.ts'],
+  onPrepare() {
+    browser.ignoreSynchronization = true;
+  }
+};
+
+
+// module.exports.config = {
+
```
Comment (amiryonatan) [inline packages/wix-ui-core/protractor.conf.js]: delete

File: packages/wix-ui-core/src/components/Input/Input.protractor.driver.ts [hunk 6]
```diff
@@ -0,0 +1,9 @@
+const inputDriverFactory = component => {
+  return {
```
Comment (amiryonatan) [inline packages/wix-ui-core/src/components/Input/Input.protractor.driver.ts]: remove return

**Pull request merged**